.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+---
+<!-- **Are you in the right place?**
+1. For issues or feature requests, please create an issue in this repository.
+2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://Rook-io.slack.com).
+3. Did you already search the existing open issues for anything similar? -->
+
+Is this a bug report or feature request?
+* Bug Report
+
+**Bug Report**
+
+What happened:
```

Comment:
I think these fields should be bold or italics to easily distinguish it from the answers at a glance.

Reply:
I would also rephrase this to be "Expected behavior" and "Deviation from expected behavior" or something similar. It just sounds more professional.